keymap_drawer/parse/qmk.py

```diff
@@ -17,7 +17,7 @@ class QmkJsonParser(KeymapParser):
     _tog_re = re.compile(r"(TG|TO|DF)\((\d+)\)")
     _mts_re = re.compile(r"([A-Z_]+)_T\((\S+)\)")
     _mtl_re = re.compile(r"MT\((\S+), *(\S+)\)")
-    _lt_re = re.compile(r"LT\((\d+), *(\S+)\)")
+    _lt_re = re.compile(r"LT\((\S+), *(\S+)\)")
```

Comment:
I am not sure I understand how this happens, can you actually have layer name defines instead of indices in keymap.json? Do you use it alongside e.g. `config.h` that contains layer defines?